gulpfile.js

```diff
@@ -22,12 +22,14 @@ const {
   integ,
   licenses,
   themeableSource,
+  bundleVendorFiles,
 } = require('./build-tools/tasks');
 
 const quickBuild = series(
   clean,
   parallel(packageJSON, generateEnvironment, generateIcons, generateIndexFile, licenses),
-  parallel(generateCustomCssPropertiesMap, styles, typescript, testUtils)
+  parallel(generateCustomCssPropertiesMap, styles, typescript, testUtils),
+  bundleVendorFiles
```

Comment:
Why do we need to bundle those in quick-build?

Reply:
In order to have the bundled dependency already inside the lib folder. We use the source in our lib folder and run our tests against, build the demo pages, etc. With that we realize quickly that some of the compiled/bundled code does not work as expected.

Reply:
Quick build includes `clean` task which removes `lib` folder. This is why running vendor task is necessary.

We can look into ways of refactoring it further, but how long is this task? If under 10 seconds, does it worth our time?

Reply:
I agree that there are ways to refactor that tasks and this may includes more modifications in the way we build the components.
The `internal/vendor` related tasks take around 5-7 sec (TS compile 5-6sec, bundling 1 sec).

Reply:
How about we build the vendor files at npm preinstall hook and on build/quick-build we copy from e.g. `vendor` to `lib/components/internal/vendor`?

Reply:
For example: https://github.com/cloudscape-design/components/pull/429

Reply:
I like this approach. There is one problem about transpiling the bundled d3-scale dependency to es15 ([see related comment in the PR](https://github.com/cloudscape-design/components/pull/429#discussion_r1009240687)).

I think with the latest update the PR is in a good shape. Let me know if this is a hard blocker for you.